From f311d3f0c58e2001b9340779de288f312aa7d4c3 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@freescale.com>
Date: Thu, 9 Jul 2015 14:37:37 +0300
Subject: [PATCH 289/451] dpaa_eth: replace devm_ioremap_prot() with
 devm_ioremap()

Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
[Xulin: Original patch taken from
Linux-LS1043A-SDK-V0.4-SOURCE-20150826-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth_base.c |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth_base.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth_base.c
index 9a0309d..cf7420e 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth_base.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth_base.c
@@ -48,6 +48,7 @@
 #include <linux/highmem.h>
 #include <linux/sort.h>
 #include <linux/fsl_qman.h>
+#include <linux/io.h>
 #include "dpaa_eth.h"
 #include "dpaa_eth_common.h"
 #include "dpaa_eth_base.h"
@@ -177,8 +178,8 @@ int dpa_bp_shared_port_seed(struct dpa_bp *bp)
 	/* allocate memory region for buffers */
 	devm_request_mem_region(bp->dev, bp->paddr,
 			bp->size * bp->config_count, KBUILD_MODNAME);
-	bp->vaddr = devm_ioremap_prot(bp->dev, bp->paddr,
-			bp->size * bp->config_count, 0);
+	bp->vaddr = devm_ioremap(bp->dev, bp->paddr,
+			bp->size * bp->config_count);
 	if (bp->vaddr == NULL) {
 		pr_err("Could not map memory for pool %d\n", bp->bpid);
 		return -EIO;
-- 
1.7.5.4

